Kaenx.Creator/MainWindow.xaml.cs

```diff
@@ -857,7 +857,12 @@ private void ClickCatalogContextRemove(object sender, RoutedEventArgs e)
         private void ClickCalcHeatmap(object sender, RoutedEventArgs e)
         {
             Models.Memory mem = (sender as Button).DataContext as Models.Memory;
-            Kaenx.Creator.Classes.MemoryHelper.MemoryCalculation(General, mem);     
+            try{
+                Kaenx.Creator.Classes.MemoryHelper.MemoryCalculation(General, mem);
+            }catch(Exception ex){
+                if(ex.Message is "out_of_memory")
```

Comment:
There is a [OutOfMemoryException](https://learn.microsoft.com/de-de/dotnet/api/system.outofmemoryexception?view=net-8.0).
Please use this instead of the Exception Message, if you want to localize the error message.
(See other PR)